File: Domain/Entities/State.cs
using System.ComponentModel.DataAnnotations;

namespace Domain.Entities
{
    public class State
    {
        [Key]
        public int StateId { get; set; }
        public int? Order { get; set; }
        public string StateName { get; set; }
        public int? ParentStateID { get; set; }
    }
}

File: client-app/src/app/models/state.ts
export interface State {
  StateId: number;
  Order: number;
  StateName: string;
  ParentStateID: number;
}

//this is only temporariliy used till i migrate to formik
export interface StateFormFormatted {
  key: number;
  value: string;
  text: string;
  stateid: number;
}
